### ISSUE TITLE
ReferenceError: __dirname is not defined

### ISSUE THREAD

**Issue by pmiguelmartins** [GreatAuk/plugin-web-update-notification#21]
In Vite Vue 3 width typescript

I'm receiving this error 

I try to use  injectFileBase but when I'm building always receive this error

How can I solve

Thansk

**Comment by GreatAuk** [comment]
vite version ?

**Comment by GreatAuk** [comment]
After version 1.2.0, in most case, you not need to set injectFileBase, it will be automatically detected from the base of vite config、publicPath of webpack config or publicPath of umi config

**Comment by vaynevayne** [comment]
`import { webUpdateNotice } from '@plugin-web-update-notification/vite'`
依然不行啊  `^1.5.2`
<img width="945" alt="image" src="https://user-images.githubusercontent.com/124666577/231946534-8f7f79c6-78a8-4375-ac56-154f70a9946a.png">

**Comment by GreatAuk** [comment]
晚上我修下。你本地 `node` 版本多少，改完我测试下。

**Comment by vaynevayne** [comment]
`v16.16.0`

**Comment by GreatAuk** [comment]
其实我是有处理 `__dirname` 的，只是打包时，target 设置错了。
```ts
export function get__Dirname() {
  if (import.meta?.url)
    return dirname(fileURLToPath(import.meta.url))
  return __dirname
}
```
可以安装下最新版本的试试 https://github.com/GreatAuk/plugin-web-update-notification/commit/74466bf88db9bea6ec5c44afd040f3f98be3d2de

**Comment by GreatAuk** [comment]
@pmiguelmartins sorry, I don't notice that you are a English user, this issue fixed in `1.5.3`.

**Comment by pmiguelmartins** [comment]
> @pmiguelmartins sorry, I don't notice that you are a English user, this issue fixed in `1.5.3`.

Already try and work just fine

thanks
